includes/Modules/AdSense.php

```diff
@@ -403,7 +403,8 @@ function( Google_Service_Adsense_Alert $alert ) {
 	protected function parse_data_response( Data_Request $data, $response ) {
 		switch ( "{$data->method}:{$data->datapoint}" ) {
 			case 'GET:accounts':
-				return array_map( array( self::class, 'filter_account_with_ids' ), $response->getAccounts() );
+				$accounts = $response->getAccounts();
+				return array_filter( array_map( array( $accounts, 'filter_account_with_state_closed' ) ) );
```

Comment:
You're still missing the `filter_account_with_ids` function as `array_map` callback. Also, it would be a bit more efficient to do the filtering first.
```suggestion
				$accounts = array_filter( $response->getAccounts(), array( self::class, 'is_account_not_closed' ) );
				return array_map( array( self::class, 'filter_account_with_ids' ), $accounts );
```

Reply:
ah, makes sense. Thank you.